### ISSUE TITLE
Line-by-line review of $Test::Builder::Level back compatibility support

### ISSUE THREAD

**Issue by rjbs** [Test-More/test-more#612]
*This is from the Berlin QAH punch list.*

ribasushi had previously committed to doing this work, or to waiving this requirement if it was not done.

**Comment by exodist** [comment]
I suspect this requirement is no longer necessary, here is why:

When the requirement was made Test-Stream worked very differently. At the time Test::Builder and Test::Stream were deeply intertwined and messed with each-others internals to some extent. As a result Test::Builder no longer used $Level internally, though it did look at it to adjust things. This is no longer the case. Test-Stream was changed to no longer consider Test::Builder at all. Test::Builder was reset back to where it was in the stable release, and refactored to live on top of Test::Stream. As a result of this $Level, and its old usage is still in place.

In other words, the changes this requirement asks to have checked no longer exists. I have no objection to someone looking over the use of $Level, but I expect it to be boring and quick since there really are not any changes from stable at this point.
